### DISCUSSION TITLE
Foreign Object

### DISCUSSION THREAD

**Discussion by henrymgarrett** [reaviz/reaflow#34] [Q&A] [answered]
Hello!

So I am trying to use a foreign object as the background for a node group parent, similar to the green node in this story: https://reaflow.dev/?path=/story/demos-nested--nested-edges .  However, when I do this, I lose the ability to create an edge from the foreign object element.

What is the recommended way to allow edge creation when a foreign object is in a node?

Thanks in advance!

Henry

**Comment by ASchwad** [comment]
Hi Henry,

atleast at the outer frame of the node (the container of the foreign object) it is possible to draw an edge, but that's very tedious and unpractical to click these few pixels.

As far as I know the code, this is currently not supported. I see currently two options: We could pass the DragEdge Method towards the foreign object and call it from there, or render the foreign object "behind" the Node container. But for a more educated answer, we should wait for @amcdnl.

**Comment by amcdnl** [comment, marked as answer]
There is a few different things you could do to make this happen:

- Disable pointer events for the foreign object
- Use ports to drag rather than the node

**Reply by henrymgarrett** [reply]
Thank you @amcdnl !  I disabled pointer-events for the foreign object and it worked perfectly.
